app/Http/Controllers/UserProjectsController.php

```diff
@@ -61,12 +62,14 @@ public function showParticipating(): Response|View
     {
         $user = Auth::user();
 
-        if ($user->context === 'organization' && $user->organization && $user->organization->isParticipant() && ($user->organization->isConsultant() || $user->organization->isConnector())) {
-            return view('projects.my-projects', [
-                'user' => $user,
-                'projectable' => $user->organization,
-                'section' => 'participating',
-            ]);
+        if (in_array($user->context, ['individual', 'organization'])) {
```

Comment:
Missed sourcing the array values from the UserContext enum. This is the third time this conditional has been checked. You may want to move it into a separate function. e.g. a shared function in this controller, a helper function, or defined in the user model.